english/hide/hide-chk.md

```diff
@@ -0,0 +1,3 @@
+you suspect the marshamllows are evil. they would like to snatch your lighter rope and fork and tie you the stake poke and burn you.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.